packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -235,6 +236,15 @@ class AppBar extends StatefulWidget implements PreferredSizeWidget {
   /// Defaults to 4, the appropriate elevation for app bars.
   final double elevation;
 
+  /// The material's shape as well its shadow.
+  ///
+  /// If shape is non null, the [borderRadius] is ignored and the material's
```

Comment:
There's no borderRadius here

Reply:
For the commentary I was inspired by the comment for the shape property in material.dart.